src/mono/wasm/Directory.Build.targets

```diff
@@ -0,0 +1,5 @@
+<Project>
```

Comment:
Do we really need this? `Directory.Build.targets` is included recursively by MSBuild. So it will go up in the directory structure until it finds the first one and stop once it find it. I think if you remove this, you will get all the repo targets and the SDK targets as well.